### ISSUE TITLE
Create Server Profile from Template

### ISSUE THREAD

**Issue by odesaphy** [HewlettPackard/terraform-provider-oneview#370]
## Scenario/Intent
When creating a Server Profile from a Template there is a wrong assignement in the resourceServerProfileCreate function (resource_server_profile.go)  

```
serverProfileByTemplate, err := config.ovClient.GetProfileTemplateByName(val.(string))  
[...]  
serverProfile = serverProfileByTemplate  <-- wrong assignment ?(objects types are different)   
```  

We should do a GET on /rest/server-profile-templates/{id}/new-profile to get a server-profile object from the server-profile-template object  
  
Because for example there is an issue between spt.serverProfileDescrition and spt.description. With the current implementation  
we endup with serverProfile.description = serverProfileByTemplate.description  
serverProfileDescription doesn't exist in the server-profile object  

## Environment
- Terraform oneview provider: v6.2
- OneView api version : 2800
## Expected Result
In the Go SDK, a new function should be implemented, for example GetNewProfileFromTemplate (GET on /rest/server-profile-templates/{id}/new-profile)  

```  
serverProfileTemplate, err := config.ovClient.GetProfileTemplateByName(val.(string))  
if err != nil || serverProfileTemplate.URI.IsNil() {  
			return err  
		}  
serverProfile, err := config.ovClient.GetNewProfileFromTemplate(serverProfileTemplate.URI)  

```  

## Actual result
If in serverProfileTemplate the description is set then in the Server Profile created from this template the sp.description = spt.description instead of sp.description=spt.serverProfileDescription  
  
a GET on /rest/server-profile-templates/{id}/new-profile to get a server profile object from template should resolve this issue

**Comment by nabhajit-ray** [comment]
Hi Olivier, The issue #369  will be fixed with OV 6.3 release. Once that is fixed this issue should also get rectified

**Comment by nabhajit-ray** [comment]
Hi Olivier, 
This issue is resolved  in the master branch. I am closing this now but you can raise another one if you think it is not resolved.
